### PR TITLE
New hana exist check

### PR DESCRIPTION
**Pull request by rhmk**
resolved conflicts from PR230

### CODE REVIEW

File: roles/sap_hana_install/README.md [hunk 1]
```diff
@@ -233,12 +233,17 @@ You can find more complex playbooks in directory `playbooks` of the collection `
 
 #### Perform Initial Checks
 
+These checks are only performed if `sap_hana_install_force` is set to `true`. Its default value is `false`
 - If variable `sap_hana_install_check_sidadm_user` is undefined or set to `y`: Check if user sidadm exists. If yes,
   abort the role.
 
-- Check if directory `/hana/shared/<sid>` exists. If yes, abort the role.
+- Check if `/usr/sap/hostctrl/exe/saphostctrl` exists and get info on running HANA instances. 
+  - If conclicting instances exist the role aborts with a failure
```
Comment (berndfinger) [inline roles/sap_hana_install/README.md:241]: typo: conclicting -> conflicting

File: roles/sap_hana_install/defaults/main.yml [hunk 2]
```diff
@@ -70,6 +70,9 @@ sap_hana_install_check_installation: no
 # no: use the command `hdblcm --action=check_installation`
 sap_hana_install_use_hdbcheck: yes
 
+# If the following variable is set to yes the idempotency check, if HANA is already installed will be skipped
```
Comment (berndfinger) [inline roles/sap_hana_install/defaults/main.yml:73]: meaning unclear, please modify.
Do you mean the following?
--->
If the following variable is set to yes, the HANA installation check will be skipped
<---

Comment (rhmk) [reply roles/sap_hana_install/defaults/main.yml:73]: yes thanks

**Comment by ja9fuchs** [comment]
**lgtm**, besides the things Bernd already commented above.

Not introduced by your update, but something to consider for further clean-up review:
- is check_mode really required to be set for modules that are not making changes (ex. stat)?
- variables `sap_hana_install_force` and `sap_hana_install_new_system` have defaults duplicated in several places, on top of their default already set in `defaults/main.yml`

**Review by ja9fuchs — APPROVED** [review]
lgtm

**Review by berndfinger — APPROVED** [review]
lgtm

**Pull request merged**